<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <addaction name="actionGradient_Mixture"/>
    <addaction name="actionOpacity"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEdit"/>
    <addaction name="actionSimple_Edit"/>
    <addaction name="actionReset"/>
    <addaction name="actionClear_Mask"/>
    <addaction name="actionAll_Mask"/>
    <addaction name="actionReverse_Mask"/>
    <addaction name="separator"/>
    <addaction name="actionEraser"/>
    <addaction name="actionPen_Size"/>
    <addaction name="actionPaint"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOption"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="text">
    <string>Edit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionGradient_Mixture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradient Mixture</string>
   </property>
  </action>
  <action name="actionClear_Mask">
   <property name="text">
    <string>Clear Mask</string>
   </property>
  </action>
  <action name="actionAll_Mask">
   <property name="text">
    <string>All Mask</string>
   </property>
  </action>
  <action name="actionOpacity">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Opacity</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionReverse_Mask">
   <property name="text">
    <string>Reverse Mask</string>
   </property>
  </action>
  <action name="actionPen_Size">
   <property name="text">
    <string>Pen Size</string>
   </property>
  </action>
  <action name="actionSimple_Edit">
   <property name="text">
    <string>Simple Edit</string>
   </property>
  </action>
  <action name="actionPaint">
   <property name="text">
    <string>Paint</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
